src/openproduct/producten/urls.py

```diff
@@ -30,11 +30,31 @@
     "TAGS": [
         {
             "name": "producten",
-            "description": "Opvragen en bewerken van PRODUCTEN.\n\n"
-            "De status opties van een PRODUCT zijn afhankelijk van de `toegestane_statussen` van het producttype.\n\n"
-            "Via `start_datum` & `eind_datum` kan de status van een product automatisch op ACTIEF en VERLOPEN worden gezet."
-            "Op het moment dat deze velden worden gezet moeten deze statussen wel zijn toegestaan op het producttype.\n\n"
-            "De status zal via de start_datum alleen naar ACTIEF veranderen mits de status INITIEEL of GEREED is. Voor de eind_datum zijn dit INTIEEL, GEREED of ACTIEF.",
+            "description": """
+## Opvragen en bewerken van PRODUCTEN.
+
+### Opmerkingen
+- Bij het aanmaken van kunnen één of meerdere eigenaren worden toegevoegd, een eigenaar moet een bsn (en/of klantnummer) of een kvk nummer (met of zonder vestigingsnummer) hebben."
+- De status opties van een PRODUCT zijn afhankelijk van de `toegestane_statussen` van het producttype.
+- Via `start_datum` & `eind_datum` kan de status van een product automatisch op ACTIEF en VERLOPEN worden gezet.
+    - Op het moment dat deze velden worden gezet moeten deze statussen zijn toegestaan op het producttype.
+    - De status zal via de start_datum alleen naar ACTIEF veranderen mits de status INITIEEL of GEREED is. Voor de eind_datum zijn dit INTIEEL, GEREED of ACTIEF.
+- `verbruiksobject` & `dataobject` zijn json velden en worden gevalideerd vanuit `vebruiksobject_schema` & `dataobject_schema` van het producttype.
+- Het veld `documenten` wordt samen met het producttype genest aangemaakt of gewijzigd.
+    - dit veld is een lijst van objecten.
+    - Bij een PUT request word de bestaande lijst overschreven met de nieuwe lijst.
+    - Bij een PATCH request wordt de lijst alleen overschreven als het veld wordt meegegeven.
```

Comment:
Not sure if `uuid` is the right field though
```suggestion
    - Bij een PATCH request wordt de lijst alleen overschreven als het veld `uuid` wordt meegegeven.
```